keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -221,8 +221,7 @@ open class SqlResourceRepository(
 
   // todo: add sql retries once we've rethought repository structure: https://github.com/spinnaker/keel/issues/740
   override fun appendHistory(event: ResourceEvent) {
-    // for historical reasons, we use the resource UID (not the ID) as an identifier in resource events
-    val uid = jooq.select(RESOURCE.UID).from(RESOURCE).where(RESOURCE.ID.eq(event.uid)).fetchOne(RESOURCE.UID)
+    val uid = jooq.select(RESOURCE.UID).from(RESOURCE).where(RESOURCE.ID.eq(event.ref)).fetchOne(RESOURCE.UID)
```

Comment:
Consider renaming `uid` to `ref` here as well (and in the parameter to `doAppendHistory()` if it's not renamed already.